### PR TITLE
Update StringTableJaUI.xml

### PR DESCRIPTION
**Pull request by Xstamper**

### CODE REVIEW

**Comment by NoA51** [comment]
Is line 152 and 153 on purpose or did you just forgot to delete it?

File: localization-pack-ja/StringTableJaUI.xml [hunk 1]
```diff
@@ -113,51 +113,51 @@
...
+            <GameDBLocalizedString>      <LocID>TOOLTIP_VIEW_FLOOR_DOWN</LocID>             		<Text>下の階層</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>TOOLTIP_VIEW_SELECT_FLOOR</LocID>             		<Text>新たな階層を追加</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>TOOLTIP_VIEW_WALLS_DOWN</LocID>             		<Text>壁をすべて隠す</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>TOOLTIP_VIEW_WALLS_AUTO</LocID>             		<Text>壁を自動的に隠す</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>TOOLTIP_VIEW_WALLS_FULL</LocID>             		<Text>すべての壁を表示する</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>TOOLTIP_VIEW_ZOOM_IN</LocID>             			<Text>拡大</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>TOOLTIP_VIEW_ZOOM_OUT</LocID>             			<Text>縮小</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>TOOLTIP_VIEW_TRANSPARENCY_LOGISTICS</LocID>         <Text>管理モードのときに半透明化しているオブジェクトを表示Show transparent objects in management mode</Text>  </GameDBLocalizedString>
```
Comment (Dysp) [inline localization-pack-ja/StringTableJaUI.xml]: @NoA51 You can create single comments/start a review by going through "files changed".

**Comment by Xstamper** [comment]
acutually I have forgotten to delete it, just ignore them.
I'll refine them when I finish translating this.

**Comment by Dysp** [comment]
Will merge, then.

**Pull request merged**